<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.core.i18n.NoSuchMessageExceptionSpec" tests="2" skipped="0" failures="0" errors="0" timestamp="2017-04-07T21:31:06" hostname="testing-gce-5d4cb8fc-d966-46b1-be13-5dd5cd7e3fa9" time="0.007">
  <properties/>
  <testcase name="Exercise constructor with errors" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.004"/>
  <testcase name="Successfully create exception" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
